low-level/sensors/impl/generic/node_hw.py

```diff
@@ -992,8 +1002,10 @@ def _parse_psu_supply_info(self, index, date, _time, sensor, sensor_num, event,
 
         sensor_id = self.sensor_id_map[self.TYPE_PSU_SUPPLY][sensor_num]
         resource_type = NodeDataMsgHandler.IPMI_RESOURCE_TYPE_PSU
+        fru_bool = self.ipmi_client.is_fru(self.fru_map[self.TYPE_PSU_SUPPLY])
```

Comment:
is_fru sounds better variable name, though no need to change current.

Reply:
ok.